components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/internal/constant/AuthenticatorAdapterConstants.java

```diff
@@ -39,6 +38,7 @@ public class AuthenticatorAdapterConstants {
     public static final String LOCAL_IDP = "LOCAL";
     public static final String FED_IDP = "FEDERATED";
     public static final String EXECUTION_STATUS_PROP_NAME = "actionExecutionStatus";
+    public static final String MULTI_VALUED_PROPERTY = "multiValued";
```

Comment:
There's a property value introduced in ClaimConstants file in claim metadata management module. Let's use that instead of using this one